Appraisals

```diff
@@ -3,29 +3,18 @@ appraise 'rails-5.2-sprockets_4' do
   gem 'sprockets', '~> 4.0.x'
   gem 'turbolinks', '~> 5'
   gem 'mini_racer', :platforms => :mri
-  gem 'therubyrhino', :platforms => :jruby
 end
 
 appraise 'rails-5.2-sprockets_3' do
   gem 'rails', '~> 5.2.x'
   gem 'sprockets', '~> 3.5'
   gem 'turbolinks', '~> 5'
   gem 'mini_racer', :platforms => :mri
-  gem 'therubyrhino', :platforms => :jruby
 end
 
-appraise 'rails-5.2_no_sprockets_webpacker_3' do
+appraise 'rails-5.2_no_sprockets_shakapacker' do
   gem 'rails', '~> 5.2.x'
```

Comment:
how much longer should we support rails 5.x?

Rails 5.2.1 Ended 11 months ago, (01 Jun 2022)

Reply:
I recommend making a yearly plan for our support for different versions of Ruby, Rails, Node, and React. Then update documentation and CI in all of our projects based on that.

It should be done in a (maximum) 30 min call with key senior developers in the team.